CHANGELOG.md

```diff
@@ -44,6 +44,7 @@
 * Add ability to mark item as Intellectual item and Restricted. Refs UIIN-1374.
 * Add permission for marking an item intellectual. Refs UIIN-1336.
 * Add permission for marking an item restricted. Refs UIIN-1335.
+* Add ability to mark an item with new statuses (In progress, In progress (non-requestable), Long missing, Unavailable, Unknown). Refs UIIN-756.
```

Comment:
This seems super petty: but can you change "In progress" to "In process", etc.? I know this is just the CHANGELOG, not the code (where it is correct), but I feel like this should be accurate.

Reply:
Exactly. My bad. Thank you for pointing this out.